### PR TITLE
LRQA-79975 Unignore CanChangePermissionsAssignedToObjectEntry test

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
There is one failed test related, I'll take a look later

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

File: modules/apps/object/object-rest-test/src/testFunctional/tests/ExposePermissionsForIndividualObjectEntries.testcase [hunk 1]
```diff
@@ -15,6 +15,8 @@ definition {
 				en_US_plural_label = "students",
 				name = "Student",
 				requiredStringFieldName = "name");
+
+			CustomObjectAPI.staticObjectDefinitionId(objectDefinitionId = ${objectDefinitionId});
```
Comment (Hijky) [inline modules/apps/object/object-rest-test/src/testFunctional/tests/ExposePermissionsForIndividualObjectEntries.testcase:19]: Could you look at this format and make sure? It seems an error.

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**